src/components/calculatorUi.js

```diff
@@ -0,0 +1,78 @@
+import React,{useState} from 'react';
+import Button from './button';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const btnFirst =['AC','+/-','%','/','7','8','9','*','4','5','6','-','1','2','3','+','0','.','=']
+  const [inputValue, setInputValue]=useState('')
+  const [numberOne,setNumberOne]=useState('')
```

Comment:
- [ ] Good job setting up every button, :+1: I noticed that every button is working properly, but I've noticed that the `+/-` button is not working as it should. As required , you should make sure that all the settled buttons are working :+1: ,Kindly fix that